### DISCUSSION TITLE
3.3.5

### DISCUSSION THREAD

**Discussion by nalexandru** [nalexandru/BlenderCompat#37] [Announcements]
Blender 3.3.5, commit [dd0cd1df1d1a](https://projects.blender.org/blender/blender/commit/dd0cd1df1d1a).
Build environment:

    Visual Studio 2019, version 16.11.25
    Windows SDK 10.0.22621
    NVIDIA CUDA 11.8
    NVIDIA OptiX 7.3

AMD HIP and SM3.0 kernel binaries copied from the official release.

<hr /><em>This discussion was created from the release <a href='https://github.com/nalexandru/BlenderCompat/releases/tag/'>3.3.5</a>.</em>

**Comment by lnesuper** [comment]
Thank you very much.
